### PR TITLE
Updated API version and Optimized DB provision validation

### PR DESCRIPTION
**Pull request by mrrajan**
This PR includes,

- API version updates for the existing payload yaml files and dbaas_utils.py
- Updates for Mongo and Crunchy validation scripts
- Optimization for DB provision validation

### CODE REVIEW

File: utils/scripts/dbaas_utils.py [hunk 1]
```diff
@@ -410,3 +415,20 @@ def add_policy_connection_nsselector(labelandconds: list):
...
+def check_instance_provision(instance, namespace):
+    """To check the provisioned instance status"""
+    oc_cli = BuiltIn().get_library_instance("OpenshiftLibrary")
+    counter = 0
+    error = None
+    while counter < 3:
+        try:
+            counter += 1
+            res = oc_cli.oc_watch("DBaaSInstance", name=instance, namespace=namespace, timeout=30)
+            if res[0]['object']['status']['phase'].lower() in ("ready", "creating"):
```
Comment (ghost) [inline utils/scripts/dbaas_utils.py:429]: I think that it's better to use the .get() when accessing to dict attributes, in order to manage different conditions and exceptions (the key does not exist, is empty...) but it's not a blocker ;)

**Review by ghost — APPROVED** [review]
LGTM

**Comment by mrrajan** [comment]
Attaching test result
[results.zip](https://github.com/RHODA-lab/rhoda-ci/files/10907156/rhoda-666-2023-03-07-4802.zip)

File: utils/scripts/dbaas_utils.py [hunk 2]
```diff
@@ -224,25 +224,28 @@ def retrieve_instances(isv, namespace):
         create_secret_cli(isv, "true")
         import_provider_account_cli(isv, namespace)
         provider_account = BuiltIn().get_variable_value(r"\${provaccname}")
-    while counter < 5:
+    while counter < 3:
```
Comment (ikanias) [inline utils/scripts/dbaas_utils.py:227]: Why is it 3 instead of 5 now?

**Review by ikanias — APPROVED** [review]
LGTM except one question

File: utils/scripts/dbaas_utils.py (hunk 2 above)
Comment (mrrajan) [reply utils/scripts/dbaas_utils.py:227]: This is on number of retries for expected status of a resource. I cut it short so we would be waiting for a minute, earlier it was more than 2 mins.

File: utils/scripts/dbaas_utils.py (hunk 1 above)
Comment (mrrajan) [reply utils/scripts/dbaas_utils.py:429]: Thanks for the suggestion, I would try to implement this part (but the constraint would be nested objects)

**Pull request merged**